### PR TITLE
TFTRT: Add more node validators

### PR DESCRIPTION
**Pull request by trevor-m**
Add validators for:

- MaxPool
- AvgPool
- FusedBatchNorm
- FusedBatchNormV2
- Pad
- Conv2D
- DepthwiseConv2DNative
- ConcatV2

Also fix a bug in test/quantization_mnist_test.py

### CODE REVIEW

File: tensorflow/contrib/tensorrt/test/quantization_mnist_test.py [hunk 1]
```diff
@@ -24,6 +24,7 @@
 # pylint: enable=unused-import
 from tensorflow.core.protobuf import config_pb2
 from tensorflow.python import data
+#from tensorflow.python.data.ops import dataset_ops
```
Comment (smit-hinsu) [inline tensorflow/contrib/tensorrt/test/quantization_mnist_test.py]: let's remove this.

**Review by smit-hinsu — DISMISSED** [review]
Thanks, Trevor! Looks good to me.

**Review by smit-hinsu — DISMISSED** [review]
Thanks!

**Comment by trevor-m** [comment]
Thanks for reviewing @smit-hinsu ! Sorry to bother you again but I made one more small change.

**Review by smit-hinsu — DISMISSED** [review]
Still looks good to me. Thanks!

**Comment by aaroey** [comment]
Thanks @trevor-m for adding these checks! Not a must for this PR, but it would be great to add test coverage for those ops.

File: tensorflow/contrib/tensorrt/convert/convert_nodes.cc [hunk 2]
```diff
@@ -2961,12 +2981,36 @@ tensorflow::Status ConvertFusedBatchNorm(OpConverterParams* params) {
...
   bool is_training = attrs.get<bool>("is_training");
   if (is_training) {
+    // Trying to use batchnorm in training mode is a very common problem.
+    // Because the error message will only be printed in VLOG(1) by the
+    // segmenter, we issue a special warning so that users will actually see it.
+    LOG(WARNING) << node_def.op() << " only supports is_training=false. If you "
+                 << "are using Keras, please call "
+                 << "keras.backend.set_learning_phase(0) before constructing "
+                 << "your model. At "
+                 << node_def.name());
```
Comment (smit-hinsu) [inline tensorflow/contrib/tensorrt/convert/convert_nodes.cc]:
```suggestion
                 << node_def.name();
```

**Comment by trevor-m** [comment]
Weird, CLA didn't like that. I will fix it in the morning.
> Co-Authored-By: trevor-m <tmorris@nvidia.com>

**Review by smit-hinsu — APPROVED** [review]

**Pull request merged**